ecommerce/src/main/resources/application.yml

```
@@ -72,3 +85,41 @@ swagger:
+      percentiles-histogram:
+        http.server.requests: true  # HTTP 요청 히스토그램
+    tags:
+      application: ${spring.application.name}  # 모든 메트릭에 애플리케이션 이름 태그 추가
+  # HikariCP 메트릭 활성화
+  hikari:
+    minimumIdle: 5
+    maximumPoolSize: 20
+    connectionTimeout: 30000
+    idleTimeout: 600000
+    maxLifetime: 1800000
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)